src/erlcloud_ddb2.erl

```diff
@@ -1450,6 +1453,21 @@ delete_item(Table, Key, Opts) ->
 %%       [{return_values, all_old},
 %%        {condition_expression, <<"attribute_not_exists(Replies)">>}]),
 %% '
+%%
+%% The ConditionExpression option can also be used in place of the legacy
+%% ConditionalOperator or Expected parameters.
```

Comment:
This example seems to match the previous, except that it uses express_attribute_names. We should just have one of these examples, or explain whatever is different between the two. (Seems like this comment might be left over from before this example was updated).